app/views/bulk_submissions/index.html.erb

```diff
@@ -22,26 +22,28 @@
           body.with_row do |row|
             row.with_cell(text: bulk_submission.created_at&.to_fs(:date_only))
             row.with_cell(text: bulk_submission.expires_at&.to_fs(:date_only))
-            row.with_cell(text: filename)
+            row.with_cell(text: govuk_link_to(filename,
+                                              bulk_submission_path(bulk_submission.id, context: :view),
+                                              method: :get,
+                                              text_colour: true))
```

Comment:
query: does the option `text_colour` accept a boolean or a string or either?

Reply:
but actually its presence with any values results in it being black/text/color

even false sets it to be text color
```ruby
govuk_link_to(filename,
  bulk_submission_path(bulk_submission.id, context: :view),
  method: :get,
  text_colour: false)
```